[FreeType] Add an implementation for FontPlatformData::familyName()

carlosgcampos opened the pull request:
#### 3db59430108ca0e3f01ec4d6ac7adbc8ec6fa7c4
<pre>
[FreeType] Add an implementation for FontPlatformData::familyName()
<a href="https://bugs.webkit.org/show_bug.cgi?id=242310">https://bugs.webkit.org/show_bug.cgi?id=242310</a>

Reviewed by Michael Catanzaro.

This will make the font name to be shown in the inspector.

* Source/WebCore/platform/graphics/FontPlatformData.cpp:
* Source/WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp:
(WebCore::FontPlatformData::familyName const):

Canonical link: <a href="https://commits.webkit.org/252129@main">https://commits.webkit.org/252129@main</a>
</pre>

mcatanzaro commented on Source/WebCore/platform/graphics/FontPlatformData.cpp:
```diff
@@ -68,7 +68,7 @@ FontPlatformData FontPlatformData::cloneWithSize(const FontPlatformData& source,
 #endif
 
-#if !USE(CORE_TEXT) && !PLATFORM(WIN)
+#if !USE(CORE_TEXT) && !PLATFORM(WIN) && !USE(FREETYPE)
```
I think all platforms are covered now, so you can probably remove this stub implementation if EWS is OK with it. Any future implementations should be able to provide a name.

carlosgcampos committed:
[FreeType] Add an implementation for FontPlatformData::familyName()
https://bugs.webkit.org/show_bug.cgi?id=242310

Reviewed by Michael Catanzaro.

This will make the font name to be shown in the inspector.

* Source/WebCore/platform/graphics/FontPlatformData.cpp:
* Source/WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp:
(WebCore::FontPlatformData::familyName const):

Canonical link: https://commits.webkit.org/252129@main
